src/screens/HomeScreen/index.tsx

```diff
@@ -1,23 +1,34 @@
 import { useNavigation } from '@react-navigation/native'
-import { Button } from 'native-base'
-
+import { Button, View } from 'native-base'
 import { StyledSafeAreaView } from 'src/styles'
+import { Ionicons } from '@expo/vector-icons'
 import { Advert } from 'src/components/Advert'
+import { IconButton } from 'src/components/IconButton'
 import { Screens } from 'src/enums'
 import { NavigationType } from 'src/types'
+import SocialLinks from 'src/features/SocialLinkGroup'
```

Comment:
Rename folder and component inside to SocialLinks

Don't make stupid, complicated names !